web/e2e/specs/flows/steps/ingestion.ts

```diff
@@ -247,6 +269,42 @@ export default function (qaProjectDir) {
       await manageFlowPage.removeFlow(flowData.flow5);
     });
 
+    it('Should verify ingest target format', async function () {
+      await manageFlowPage.createFlow(flowData.flow1);
+      await editFlowPage.addStep(flowData.flow1, json);
+      await browser.sleep(5000);
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"json\"");
+      await ingestStepPage.setTargetFileType("XML");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"xml\"");
+      await ingestStepPage.setTargetFileType("Text");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"text\"");
+      await ingestStepPage.setTargetFileType("Binary");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"binary\"");
+    });
+
+    it('Should verify target permissions', async function () {
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-output_permissions \"rest-reader,read,rest-writer,update\"");
+      await ingestStepPage.targetPermissions.clear();
+      await ingestStepPage.targetPermissions.sendKeys("rest-reader,read,rest-writer,update,user1,execute");
+      await ingestStepPage.targetPermissions.sendKeys(protractor.Key.ENTER);
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-output_permissions \"rest-reader,read,rest-writer,update,user1,execute\"");
+    });
+
+    it('Should target uri replacement', async function () {
+      await ingestStepPage.targetUriReplace.clear();
+      await ingestStepPage.targetUriReplace.sendKeys("/web/e2e/qa-project/");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-output_uri_replace \"/web/e2e/qa-project/\"");
+      await ingestStepPage.targetUriReplace.clear();
+    });
+
+    it('Should uri preview', async function () {
+      await ingestStepPage.targetUriReplace.clear();
+      await ingestStepPage.targetUriReplace.sendKeys("/C/dev/fork/marklogic-data-hub/web/e2e/qa-project/input/flow-test/json/, '/web/e2e/qa-directory/'");
```

Comment:
not sure why this is not failing on jenkins? Seems like a local directory path. It's better to use the variable qaProjectDir

examples: 
await ingestStepPage.setInputFilePath(qaProjectDir + "/input/advantage");
await ingestStepPage.setTargetUriReplace(qaProjectDir + "/input/advantage" + ", " + "\'\/advantage\'");

Reply:
It won't fail on Jenkins because it's not testing the correctness of the path which will be different on Mac/Unix and Windows but verifies that the path parameter displays on MLCP.

Reply:
I see, it's for the target uri preview